tests/scripts/all.sh

```diff
@@ -2564,13 +2571,28 @@ component_test_psa_crypto_config_accel_all_curves_except_p192 () {
     not grep mbedtls_ecdh_ library/ecdh.o
     not grep mbedtls_ecdsa_ library/ecdsa.o
     not grep mbedtls_ecjpake_ library/ecjpake.o
+    if [ $BUILTIN_CURVE == "MBEDTLS_ECP_DP_SECP192R1_ENABLED" ]; then
+        # The only built-in curve is Short Weierstrass, so ECP shouldn't have support for Montgomery curves.
+        # Functions with mxz in their name are specific to Montgomery curves.
+        not grep mxz library/ecp.o
+    else
```

Comment:
This assume that if it's not `MBEDTLS_ECP_DP_SECP192R1_ENABLED` then it's `CURVE25519`, which I think should be either tested or documented. Also, please add a comment as above saying that `ecp_muladd` is specific to Short Weierstrass curves.